tests/VirtoCommerce.NotificationsModule.Tests/UnitTests/LiquidTemplateRendererUnitTests.cs

```diff
@@ -152,6 +153,51 @@ public async Task RenderAsync_ContextHasLayoutId_LayoutRendered()
             Assert.Equal("header test_content footer", result);
         }
 
+        [Fact]
+        public async Task RenderAsync_ContextHasLayoutId_ArrayContent_LayoutRendered()
+        {
+            var layoutId = Guid.NewGuid().ToString();
+            var content = @"{% capture content %} Order Number: {{customer_order.number }}, Items Catalog Name: {{ var1 = customer_order.items | where: 'CatalogId' '==' '1' | array.first }} {{ var1.name }} {% endcapture %}";
+            
+            //assign var1 = item.series.properties | where: ‘name' ' == ' 'UnitsPerPackage' | array.first }}{ { var1.catalog_id } }
+
+            dynamic model = new
+            {
+                CustomerOrder = new CustomerOrder
+                {
+                    Number = "123456",
+                    Total = 3,
+                    Items = new[]
+                    {
+                        new LineItem{CatalogId = "1",Name = "Item1"},
+                        new LineItem{CatalogId= "2",Name = "Item2"},
+                        new LineItem{CatalogId = "1",Name = "Item3"},
+                    }
+                }                
+            };
+            
+            var layout = new NotificationLayout
+            {
+                Id = layoutId,
+                Template = "header {{content}} footer",
+            };
+
+            _notificationLayoutServiceMock
+                .Setup(x => x.GetAsync(It.Is<IList<string>>(x => x.FirstOrDefault() == layoutId), It.IsAny<string>(), It.IsAny<bool>()))
+                .ReturnsAsync(new[] { layout });
+
+            var context = new NotificationRenderContext
+            {
+                Template = content,
+                LayoutId = layoutId,
+                Model = model
+            };
+            
+            var result = await _liquidTemplateRenderer.RenderAsync(context);
+            Assert.Equal("header  Order Number: 123456, Items Catalog Name:  Item1  footer", result);
```

Comment:
Add tests for operations other than `==`.

Reply:
fixed, I added a new test using greater than

Reply:
Not fixed, there are more operations